src/SIL.XForge.Scripture/ClientApp/src/app/app.component.spec.ts

```diff
@@ -576,6 +566,17 @@ describe('AppComponent', () => {
       verify(mockedUserService.editDisplayName(false)).once();
     }));
 
+    it('non auth0 connection users can edit name', fakeAsync(() => {
```

Comment:
The phrasing "non auth0 connection" is kind of a misnomer; these users are connected via Auth0. What about specifically calling out Paratext users for this test?